### PR TITLE
KATI-42_Patient-Appointment-List-UI

### PR DESCRIPTION
**Pull request by vtnlinh95**

### CODE REVIEW

File: app/src/main/java/com/kms/cura/utils/CurrentUserProfile.java [hunk 1]
```diff
@@ -1,18 +1,25 @@
...
+import java.util.List;
 
 /**
  * Created by linhtnvo on 6/8/2016.
  */
 public class CurrentUserProfile {
     private static CurrentUserProfile mInstance;
     private UserEntity entity;
+    private List<AppointmentEntity> upcomingAppts;
+    private List<AppointmentEntity> pastAppts;
```
Comment (hieumai) [inline app/src/main/java/com/kms/cura/utils/CurrentUserProfile.java]: Don't save this fields here

File: com.kms.cura.entity/src/com/kms/cura/entity/user/UserEntity.java [hunk 2]
```diff
@@ -1,5 +1,8 @@
 package com.kms.cura.entity.user;
 
+import java.util.List;
+
+import com.kms.cura.entity.AppointmentEntity;
```
Comment (hieumai) [inline com.kms.cura.entity/src/com/kms/cura/entity/user/UserEntity.java]: remove un-used imports

File: com.kms.cura.entity/src/com/kms/cura/entity/user/AdminUserEntity.java [hunk 3]
```diff
@@ -1,9 +1,14 @@
 package com.kms.cura.entity.user;
 
+import java.util.List;
+
+import com.kms.cura.entity.AppointmentEntity;
```
Comment (hieumai) [inline com.kms.cura.entity/src/com/kms/cura/entity/user/AdminUserEntity.java]: Remove un-used imports

File: app/src/main/java/com/kms/cura/utils/CurrentUserProfile.java (hunk 1 above)
Comment (vtnlinh95) [inline app/src/main/java/com/kms/cura/utils/CurrentUserProfile.java]: @hieumai so where I should save these fields ?

**Comment by vtnlinh95** [comment]
@hieumai fixed

**Comment by hieumai** [comment]
OK from me. @huuduy216  and @toannguyen71195  please start verifying this

**Comment by hieumai** [comment]
@vtnlinh95  please rebase this so I can merge

**Comment by hieumai** [comment]
@vtnlinh95 solve conflicts

**Comment by vtnlinh95** [comment]
@hieumai rebased

**Comment by hieumai** [comment]
@vtnlinh95 please solve conflicts again

**Pull request merged**